services/docker-compose.yml

```diff
@@ -278,6 +278,9 @@ services:
       - RABBIT_PORT=${RABBIT_PORT}
       - RABBIT_SECURE=${RABBIT_SECURE}
       - RABBIT_USER=${RABBIT_USER}
+      - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED}
```

Comment:
MINOR: keep all env var names sorted alphabetically please :-) It helps spotting duplicates and finding issues with these huge amount of envs